docs/computing/runtimes/index.md

```diff
@@ -55,20 +57,52 @@ Use these in your custom runtime by copying them to `/rootfs/sbin/init` and
 of the capabilities provided by the aleph.im ecosystem, using the default of your distribution 
 (ex: [systemd](https://systemd.io/), [OpenRC](https://github.com/OpenRC/openrc), ...) should work as well.
 
-### List of official runtimes
+## List of official runtimes
+
+
+
+### Functions
+Runtime used for functions
+
+| name                                                         | hash                                                                | filesystem |
+|--------------------------------------------------------------|---------------------------------------------------------------------|------------|
+| Official runtime with Debian 11, Python 3.9 & Node.js 14     | bd79839bf96e595a06da5ac0b6ba51dea6f7e2591bb913deccded04d831d29f4    | ext4       |
```

Comment:
I can't find this runtime being used in the current code of the console

Reply:
it is used in the python client